### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Pherpher089-zz**
Set up Home Page with skeleton components

Fixes # (issue)
files structure
css issues
component hierarchy 

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [xI have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: groovin-music/src/components/Home/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <div className="homepage-container">
+      {/* add nav bar here */}
+
+      <div className="sidebar">
+        <Sidebar />
+      </div>
+
+      <div className="main">
+        <Route exact path="/" component={HomeMain} />
+        <Route path="/playlist-page" component={PlaylistPage} />
```
Comment (alemasney) [inline groovin-music/src/components/Home/Homepage.js]: nice work on the routes

File: groovin-music/src/index.js [hunk 2]
```diff
@@ -24,30 +24,29 @@ const HELLO_QUERY = gql`
...
+//   <ApolloProvider client={client}>
+//       <div>
+//           <h2>Connected FE to BE</h2>
+//            <Homepage />
+//           {/* <Hello></Hello> */}
+//       </div>
+//   </ApolloProvider>
+
+ReactDOM.render(
+  <Router>
```
Comment (alemasney) [inline groovin-music/src/index.js]: Nice.

**Review by alemasney — APPROVED** [review]
very clean file structure, also all components render correctly with the nav links.

great job!

**Review by jjamaltwin — APPROVED** [review]
Components and file structure looks good! Great Job!

**Pull request merged**